spec/controllers/market_lists_controller_spec.rb

```diff
@@ -0,0 +1,87 @@
+require 'rails_helper'
+
+RSpec.describe MarketListsController, type: :controller do
+  describe 'GET #index' do
+    it 'assigns @market_lists ordered by market_date' do
+      market_list1 = MarketList.create(name: 'Lista 1', market_date: Date.today)
+      market_list2 = MarketList.create(name: 'Lista 2', market_date: Date.yesterday)
+
+      get :index
+
+      expect(response).to be_successful
```

Comment:
It would be better if you could be more specific with http responses, with all tests, like this

```suggestion
      expect(response).to have_http_status(:ok)
```